packages/docs/docs/integration-frameworks/angular.mdx

```diff
@@ -0,0 +1,276 @@
+:::
+
+  </TabItem>
+</Tabs>
+
+## Configure TailwindCSS (optional)
+
+If your project uses **TailwindCSS**, ensure that it applies styles to `.tsx` files by updating the `tailwind.config.js` file as follows:
+
+```js twoslash title="tailwind.config.js"
+module.exports = {
+  content: [
+    './src/**/*.{html,ts,tsx}',  // Add .tsx files to Tailwind content paths
+    // other paths
+  ],
+  // other Tailwind configurations
+};
+```
+
+You should also install the following package:
+
+<Tabs
+defaultValue="npm"
+values={[
+{ label: 'npm', value: 'npm', },
+{ label: 'pnpm', value: 'pnpm', },
+{ label: 'yarn', value: 'yarn', },
+{ label: 'bun', value: 'bun', },
+]
+}>
+<TabItem value="npm">
+
+```bash title="Use npm as the package manager"
+npm i @remotion/tailwind
+```
+
+  </TabItem>
+  <TabItem value="pnpm">
+
+```bash title="Use pnpm as the package manager"
+pnpm i @remotion/tailwind
+```
+
+  </TabItem>
+
+  <TabItem value="yarn">
+
+```bash title="Use Yarn as the package manager"
+yarn add @remotion/tailwind
+```
+
+  </TabItem>
+
+  <TabItem value="bun">
+
+```bash title="Use Bun as the package manager and runtime"
+bun i @remotion/tailwind
+```
+
+:::note
+Bun as a runtime is mostly supported. [Read more here](/docs/bun).
+:::
+
+  </TabItem>
+</Tabs>
+
+## Create a Remotion folder
+
+For better separation, create a folder to hold your Remotion files:
+
+```plaintext
+src/app/remotion
+```
+
+Copy the contents of your Remotion project or a starter template (e.g., HelloWorld) into this new folder. This will help separate Remotion related files from the rest of your Angular codebase.
+
+## Copy `remotion.config.ts`
+
+Copy the `remotion.config.ts` file to the root directory of your Angular project, placing it at the same level as `tailwind.config.js` or `package.json`.
+
+This configuration file is necessary for Remotion to recognize and compile your project settings.
+
+## Configure TypeScript for JSX
+
+To enable JSX support in Angular, update the `tsconfig.json` file by setting `"jsx": "react"` under `compilerOptions`. This configuration allows Angular to interpret JSX syntax used in Remotion's React components.
+
+```json title="tsconfig.json"
+{
+  "compilerOptions": {
+    "jsx": "react",
+    "skipLibCheck": true,  // Recommended to avoid errors with certain libraries
+    // other options
+  }
+}
+```
+
+:::note
+The "skipLibCheck": true setting is also recommended to prevent compatibility issues with certain library types.
+:::
+
+## Create a React wrapper component for Angular
+
+To embed Remotion components in Angular, create a **wrapper component**:
+
+1. In your `remotion` folder, create a file named `PlayerViewWrapper.tsx`.
+2. Ensure each `.tsx` file imports React explicitly at the top of the file:
+
+```tsx twoslash title="PlayerViewWrapper.tsx"
+import {AfterViewInit, Component, effect, ElementRef, Input, OnDestroy, signal, Signal, ViewChild, WritableSignal} from '@angular/core';
```

Comment:
If you want to use `twoslash` to typecheck the snippets here, you need to add the dependencies (`@angular/core`) to the package.json of `packages/docs` as well